elasticdl/python/master/main.py

```diff
@@ -106,6 +107,14 @@ def main():
     model_inst = load_model_from_module(
         args.model_def, model_module, args.model_params
     )
+    if not model_inst.trainable_variables:
```

Comment:
Function API initializes parameters at once. And subclass API will call `build` method to initialize parameters.